Solutions/README.md

```diff
@@ -6,36 +6,38 @@ Azure Sentinel Solutions provide an in-product experience for central discoverab
 * Customers can easily deploy and optionally enable content to get started immediately.
 * Providers or partners can deliver combined product or domain or vertical value via Solutions in Azure Sentinel and be able to productize investments too.
 
-# Solution Package
-Clone the repository [Sentinel-AzMP-Solutions](https://msazure.visualstudio.com/One/_git/Sentinel-AzMP-Solutions) to C:\One.
+# Creating Solution Package
+**Note: Packaging tool is currently not available. it will be available in the near future.**
+
+Clone the repository [Azure-Sentinel](https://github.com/Azure/Azure-Sentinel) to C:\One.
 ## Create Input File
-Create an input file and place it in the path C:\One\Sentinel-AzMP-Solutions\solutions\automation\input.
+Create an input file and place it in the path C:\One\Azure-Sentinel\Tools\PowerShell\Create-Sentinel-Solution\input.
 
 **Input File Format:**
 ```
 /**
- * Solution Automation Input File Interface
+ * Solution Automation Input File Json
  * -----------------------------------------------------
- * The purpose of this interface is to provide detail on
+ * The purpose of this json is to provide detail on
  *  the various fields the input file can have.
  */
-interface SolutionAutomationInput {
-  Name: string;                //Solution Name      - Ex. "Symantec Endpoint Protection"
-  Author: string;              //Author of Solution - Ex. "Eli Forbes - v-eliforbes@microsoft.com"
-  Logo: string;                //Link to the Logo used in the CreateUiDefinition.json
-  Description: string;         //Solution Description used in the CreateUiDefinition.json
-  WorkbookDescription: string; //Workbook description from ASI-Portal Workbooks Metadata
-
+{
+  "Name": "{SolutionName}",                   //Solution Name      - Ex. "Symantec Endpoint Protection
+  "Author": "{AuthorName - id}",              //Author of Solution - Ex. "Amarnath Pamidi - v-ampami@microsoft.com"
+  "Logo": "{logolink}",                       //Link to the Logo used in the CreateUiDefinition.json
+  "Description": "{ Solution Description}",   //Solution Description used in the CreateUiDefinition.json
+  "WorkbookDescription": "{Description of workbook}", //Workbook description from ASI-Portal Workbooks Metadata
```

Comment:
This field can be either an array or string, depending on if we want multiple workbooks with differing descriptions.